pkg/flink/handler.go

```diff
@@ -158,10 +158,16 @@ func (h flinkResourceHandler) OnAbort(ctx context.Context, tCtx pluginsCore.Task
 	return abortBehavior, nil
 }
 
-func FlinkClusterTaskLogs(ctx context.Context, flinkClusterName string, flinkClusterNamespace string) ([]*core.TaskLog, error) {
+type FlinkTaskLogs struct {
+	ClusterName string
+	Namespace   string
+	Config      *Config
+}
+
+func FlinkClusterTaskLogs(ctx context.Context, flinkTaskLogs FlinkTaskLogs) ([]*core.TaskLog, error) {
```

Comment:
Let's have `*Config` as an argument, and could `FlinkTaskLogsInput` be a better name? similar to `tasklog.Input`?